packages/beacon-node/test/unit/db/api/repositories/dataColumn.test.ts

```diff
@@ -53,7 +53,7 @@ describe("block archive repository", function () {
     singedBlock.message.body.blobKzgCommitments.push(commitment);
     singedBlock.message.body.blobKzgCommitments.push(commitment);
     singedBlock.message.body.blobKzgCommitments.push(commitment);
-    const dataColumnSidecars = computeDataColumnSidecars(config, singedBlock, {
+    const dataColumnSidecars = getDataColumnSidecars(config, singedBlock, {
```

Comment:
i would somehow prefer compute because thats what its doing since all relevant data is passed into the args, so its not a class/object/api from which we are getting